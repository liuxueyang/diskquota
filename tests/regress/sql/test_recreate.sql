\c

CREATE DATABASE test_recreate;

\c diskquota

INSERT INTO diskquota_namespace.database_list(dbid) SELECT oid FROM pg_database WHERE datname = 'test_recreate';

\c test_recreate
CREATE EXTENSION diskquota;
SELECT diskquota.wait_for_worker_new_epoch(); -- shoud be ok
SELECT diskquota.pause();
SELECT diskquota.wait_for_worker_new_epoch();
DROP EXTENSION diskquota;

\c contrib_regression
DROP DATABASE test_recreate;
