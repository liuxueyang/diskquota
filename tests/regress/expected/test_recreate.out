\c
CREATE DATABASE test_recreate;
\c diskquota
INSERT INTO diskquota_namespace.database_list(dbid) SELECT oid FROM pg_database WHERE datname = 'test_recreate';
\c test_recreate
CREATE EXTENSION diskquota;
SELECT diskquota.wait_for_worker_new_epoch(); -- shoud be ok
 wait_for_worker_new_epoch 
---------------------------
 t
(1 row)

SELECT diskquota.pause();
 pause 
-------
 
(1 row)

SELECT diskquota.wait_for_worker_new_epoch();
 wait_for_worker_new_epoch 
---------------------------
 t
(1 row)

DROP EXTENSION diskquota;
\c contrib_regression
DROP DATABASE test_recreate;
